script.js

```diff
@@ -10,16 +9,20 @@ function leftPad(number, targetLength) {
     return output;
 }
 
-function check(combo){
-	
+function check(count){
+ return count === 49
```

Comment:
In this exercise we are using the `pinCode` variable as a static value and *pretending* like we don't know what it is in the rest of our code. The idea is that we can use our for loops to iterate through all possible pin combinations and guess the right code regardless of what it is set to.

With this line of code, your `check` method will only behave correctly if `pinCode` is the number `49`. There are 2 issues with this:
1. You're trying to compare a number: `49` to a string `0049`
2. Use `==` for testing equality

Modify your `check` method to use look at the `pinCode` variable instead of a static value, correct your conditional logic and resubmit.